### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by lindsaymarkward**
Check formatting, naming and logic
Thank you @dmitrykonovalovjcueduau

### CODE REVIEW

File: oddName.py [hunk 1]
```diff
@@ -2,10 +2,24 @@
 Lindsay Ward
 """
 
-name = input("Name: ")
-while name == "":
-    print("Name can not be blank")
+
+def main():
+    name = get_name()
```
Comment (dmitrykonovalovjcueduau) [inline oddName.py:7]: look good

File: oddName.py [hunk 2]
```diff
@@ -2,10 +2,24 @@
...
+def print_characters(string, number):
+    for i in range(0, len(string), number):
+        print(string[i])
+
+
+def get_name():
     name = input("Name: ")
-for i in range(0, len(name), 2):
-    print(name[i])
+    while name == "":
```
Comment (dmitrykonovalovjcueduau) [inline oddName.py:19]: nice

Comment (lindsaymarkward) [inline oddName.py:19]: Yes, I thought so!

Comment (dmitrykonovalovjcueduau) [inline oddName.py:19]: ok

File: variableFormatting.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+
+# Word Counter
+# filename = input("Enter the filename: ")
+filename = "README.md"
+inputFile = open(filename, 'r')
+# initialise a dictionary of unique words
```
Comment (dmitrykonovalovjcueduau) [inline variableFormatting.py]: not necesary

**Pull request merged**